src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1019,6 +1019,8 @@ protected DynamoModel(IStartConfiguration config)
             State = DynamoModelState.StartedUIless;
             // This event should only be raised at the end of this method.
             DynamoReady(new ReadyParams(this));
+            // Write index to disk only once
+            LuceneUtility.CommitWriterChanges();
```

Comment:
I think at this moment all the packages nodes were already indexed and added to `DynamoModel.SearchModel.SearchEntries` 
then I think we are good to go, we just need to test the next scenarios when the packages are loaded from Dynamo Startup process and when we install a package with PackageManagerSearch (validate that the nodes are indexed correctly).
Thanks

Reply:
I confirmed those locally but will need @avidit 's testing after this is merged